### ISSUE TITLE
Can't run dev server via ./manage.py runserver

### ISSUE THREAD

**Issue by leksuss** [hseritt/repositext#38]
Hello!

I have a trouble with running server with sqlite. What I do according your instructions:
 -  create `local.py` with `project_settings = 'repositext.settings-local` string in root
 - create `settings-local.py` in `repositext` folder with default copied settings from `settings.py` except `DATABASES` block. This block I replaced with this:
```
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.sqlite3',
        'NAME': 'repositext.sqlite',
    }
}
```

But when I run:
```
 ./manage.py runserver
```
it's still trying to use `mysql`. 

But if I say directly what settings file to use:
```
./manage.py runserver 127.0.0.1:8000 --settings=repositext.settings-local
```
it's successfully start.

Maybe I do something wrong?

**Comment by hseritt** [comment]
Hi Audrey, this is because the standard settings.py has MySQL set. If you
do just ./manage.py runserver it will use settings.py. To run ./manage.py
runserver with other settings file you should do: ./manage.py runserver
--settings repositext.[my_settings module] ... hopefully that helps.

Now, in order to use the settings-local.py you would use runserver.sh or
reset-data.sh. Unfortunately to run ./runtests.sh you will still need to
set the specific settings file you wish to use. Hope that helps.

Thanks,
Harlin

**Comment by leksuss** [comment]
Thank you for your answer. I think this is a good idea to add this information to readme. And as I understand for any usage manage.py (like makemigrations, migrate) we should set local settings via --settings flag.

**Comment by hseritt** [comment]
Right. This is just an extra option. Nothing in the README implies has to
use it.
